docusaurus/docs/iOS/05-ui-cookbook/21-session-timers.mdx

```diff
@@ -0,0 +1,287 @@
+---
+title: Session Timers
+description: Documentation on using session timers.
+---
+
+# Session Timers
+
+In some cases, you want to be able to limit the duration of a call. StreamVideo supports this use-case by allowing you to specify the duration of a call on creation. Additionally, it provides you a way to extend the duration during the call, if needed.
+
+## Low-level client capabilities
+
+First, let's see how we can create a call that has limited duration.
+
+```swift
+let call = streamVideo.call(callType: "default", callId: "callId")
+try await call.create(members: [], maxDuration: 300)
+```
+
+This code will create a call, which will have a duration of 300 seconds (5 minutes), as soon as the session is started (a participant joined the call). 
+
+You can check the start date of a call with the following code:
+
+```swift
+let startedAt = call.state.session?.startedAt
+```
+
+When the `maxDuration` of a call is specified, the call session also provides the `timerEndsAt` value, which provides the date when the call will end. When a call is ended, all the participants are kicked out of the call.
+
+```swift
+let timerEndsAt = call.state.session?.timerEndsAt
+```
+
+## Extending the call duration
+
+You can also extend the duration of a call, both before or during the call. To do that, you should use the `call.update` method:
+
+```swift
+let newDuration = (call.state.settings?.limits.maxDurationSeconds ?? 0) + Int(extendDuration)
```

Comment:
Note: Apple uses `XYZInSeconds` quite a bit but that is already existing API so ignore the comment

Reply:
this is from the generated open api code, so we shouldn't change it.